Type error

Hi 👋 

There is an issue with typing in your preset.
When importing `unocss-preset-grid`, we get following typescript error:
`Cannot find module 'unocss-preset-grid' or its corresponding type declarations.`
This is due to your package.json configuration that declares `"types": "./dist/index.d.ts"` without anything creating this index.d.ts on build.
Can you fix this?

Thanks in advance

--- a/package.json
+++ b/package.json
@@ -31,7 +31,8 @@
   "devDependencies": {
     "typescript": "^5.1.6",
     "unocss": "^0.53.4",
-    "vite": "^4.4.2"
+    "vite": "^4.4.2",
+    "vite-plugin-dts": "^3.4.0"
   },
   "peerDependencies": {
     "unocss": "^0.53.4"

--- a/vite.config.ts
+++ b/vite.config.ts
@@ -1,5 +1,6 @@
 import { defineConfig } from 'vite'
 import unocss from 'unocss/vite'
+import dts from 'vite-plugin-dts'
 
 import { presetGrid } from './src'
 
@@ -10,6 +11,7 @@ export default defineConfig({
         presetGrid(),
       ],
     }),
+    dts(),
   ],
   build: {
     lib: {

